wazo_call_logd/generator.py

```diff
@@ -154,10 +159,29 @@ def from_cel(self, cels):
             call_logs_to_delete=call_logs_to_delete,
         )
 
-    def call_logs_from_cel(self, cels):
+    def call_logs_from_cel(self, cels: list[CEL]) -> list[CallLog]:
         result = []
-        for _, cels_by_call_iter in self._group_cels_by_linkedid(cels):
-            cels_by_call = list(cels_by_call_iter)
+        for linkedids, cels_by_call in self._group_cels_by_shared_channels(cels):
+            logger.debug(
+                'interpreting %d cels from correlated linkedids(%s)',
+                len(cels_by_call),
+                linkedids,
+            )
+
+            terminated_links = set(
+                cel.linkedid
+                for cel in cels_by_call
+                if cel.eventtype == CELEventType.linkedid_end
+            )
+            unterminated_links = set(
+                linkedid for linkedid in linkedids if linkedid not in terminated_links
+            )
```

Comment:
Suggestion: 
```
unterminated_links = set(linkedids) - terminated_links
```
And maybe check `if len(linkedids) != len(unterminated_links)` first, then compute the difference for logging only.